explorer/trace_testdata/full_trace.carbon

```diff
@@ -93,18 +93,18 @@ fn Main() -> i32 {
 // CHECK:STDOUT: checking BindingPattern n: i32
 // CHECK:STDOUT: checking ExpressionPattern i32
 // CHECK:STDOUT: checking IntTypeLiteral i32
-// CHECK:STDOUT: (+) stack-push: i32 .0.
-// CHECK:STDOUT: (+) stack-push: i32 .0.
+// CHECK:STDOUT: (+) stack-push: pos-0 ValueExpressionAction `i32` (full_trace.carbon:11)
```

Comment:
`pos-0` isn't a very intuitive notation to me. How about something like `pos: 0` for consistency with things like `results: ...`

Reply:
Yes I think its reasonable to go with `pos: <num>` for consistency.